raf.go

```diff
@@ -0,0 +1,79 @@
+// Extracted from github.com/oskca/gopherjs-raf
```

Comment:
I think we should not pull this in. Instead, let's just use the browser's native RAF.

GopherJS already doesn't support old browsers unless they have support for typed arrays, and according to caniuse.com the browser support for typed arrays and RAF is identical with exception of Opera Mini (which [doesn't support `setTimeout` either](https://en.wikipedia.org/wiki/Opera_Mini#JavaScript_support)):

- http://caniuse.com/#feat=typedarrays
- http://caniuse.com/#feat=requestanimationframe

The main benefits of not pulling this in are:

1. Less bytes sent to the browser.
1. Our `LICENSE` file doesn't need to explicitly mention this file as an outlier.

Reply:
Logic seems sound enough.  Do you want to export the RAF wrappers?

Reply:
Let's not for now.